guides/common/modules/proc_applying-remote-scap-resources-in-a-disconnected-environment.adoc

```diff
@@ -28,19 +23,15 @@ which is referenced from datastream
 # oscap info _/usr/share/xml/scap/ssg/content/ssg-rhel8-ds.xml_
 ...
 		Referenced check files:
-			ssg-rhel8-oval.xml
-				system: http://oval.mitre.org/XMLSchema/oval-definitions-5
 			ssg-rhel8-ocil.xml
 				system: http://scap.nist.gov/schema/ocil/2
-			*security-data-oval-com.redhat.rhsa-RHEL8.xml.bz2*
-				system: http://oval.mitre.org/XMLSchema/oval-definitions-5
```

Comment:
The procedure will not make sense without these lines.

Reply:
However, since RH OVAL content v1 is deprecated and was replaced by v2 in newest SSGs, we might update the examples for v2, such as "security-data-oval-**v2**-RHEL8-rhel-8.oval.xml.bz2"